semantic-ast/semantic-ast.cabal

```diff
@@ -47,8 +47,14 @@ library
                      , bytestring ^>= 0.10.8.2
                      , optparse-applicative >= 0.14.3 && < 0.16
                      , pretty-simple ^>= 3.1.0.0
+                     , aeson                ^>= 1.4.2.0
+                     , text                 ^>= 1.2.3.1
+                     , bytestring           ^>= 0.10.8.2
+                     , aeson-pretty         ^>= 0.8.8
   hs-source-dirs:      src
   default-language:    Haskell2010
+  ghc-options:       -Wall
+                     -Wredundant-constraints
```

Comment:
I know `Wredundant-constraints` is a warning not included in `Wall` and therefore added separately, and I understand it is triggered when multiple constraints are provided in a type signature where one subsumes the other, but I forgot why @patrickt recommended I should add this for the goals of the `semantic-ast` package? Or is it just a convention/approach we take across all packages and we want to be consistent? sorry for the yak-shave-y question

Reply:
ah right